pom.xml

```diff
@@ -73,22 +109,34 @@
             </properties>
         </profile>
         <profile>
-            <id>mult-div-suite</id>
+            <id>hw1-mult-div-suite</id>
             <properties>
                 <testng.suite.file>${testng.suite.path}/hw1/hw1-mult-div-suite.xml</testng.suite.file>
             </properties>
         </profile>
         <profile>
-            <id>sqrt-suite</id>
+            <id>hw1-sqrt-suite</id>
             <properties>
                 <testng.suite.file>${testng.suite.path}/hw1/hw1-sqrt-suite.xml</testng.suite.file>
             </properties>
         </profile>
         <profile>
-            <id>all-tests-suite</id>
+            <id>hw1-all-tests-suite</id>
             <properties>
                 <testng.suite.file>${testng.suite.path}/hw1/hw1-all-tests-suite.xml</testng.suite.file>
             </properties>
         </profile>
+        <profile>
+            <id>hw2-all-tests-suite</id>
+            <properties>
+                <testng.suite.file>${testng.suite.path}/hw2/hw2-all-tests-suite.xml</testng.suite.file>
+            </properties>
+        </profile>
+        <profile>
+            <id>hw3-all-tests-suite</id>
+            <properties>
+                <testng.suite.file>${testng.suite.path}/hw3/hw3-all-tests-suite.xml</testng.suite.file>
+            </properties>
+        </profile>
     </profiles>
```

Comment:
Why there is no profile has been added for hw5?

Reply:
added